lib/containers/urls.pm

```diff
@@ -51,64 +51,211 @@ sub get_opensuse_registry_prefix {
     }
 }
 
+our %registries = (
+    sle => {
+        '12-SP3' => {
+            released_registry => sub { "registry.suse.com/suse/sles12sp3" },
```

Comment:
I think the hash is already about registries, so lets call it `released` `test`, wdyt?
```suggestion
            released => sub { "registry.suse.com/suse/sles12sp3" },
```

Reply:
could be. but i would like to keep the `totest`. we used it even in verbal communication when we talk about images and it is less general than `test`. is it ok?

Reply:
sure!

Reply:
*this change is ready for review*